### PR TITLE
New iOS timer screen design

### PR DESCRIPTION
**Pull request by amulware**

### CODE REVIEW

File: Ross/Info.plist [hunk 1]
```diff
@@ -56,6 +56,8 @@
 	<key>UIAppFonts</key>
 	<array>
 		<string>Fonts/icomoon.ttf</string>
+		<string>Fonts/SF-UI-Text-Light.otf</string>
+		<string>Fonts/SF-UI-Text-Regular.otf</string>
```
Comment (monday8am) [inline Ross/Info.plist]: I think we can't use San Francisco font in that way. Maybe is better to use the *system font* utility.
`v.Font = UIFont.SystemFontOfSize(14f, UIFontWeight.UltraLight);`

http://stackoverflow.com/questions/33096954/san-francisco-font-in-ios-8

Comment (amulware) [inline Ross/Info.plist]: However, this means that for older versions of iOS the app will look different, right? Are we ok with this?

Comment (monday8am) [inline Ross/Info.plist]: they will use their default system font. I'm afraid if we include the SF font without the proper license, Apple rejects the binary. But we can take the risk if you want.

**Comment by amulware** [comment]
@monday8am consider this a resubmit. As far as I'm concerned we can merge this.

**Comment by monday8am** [comment]
thanks @amulware, I'll review it ASAP 👍

**Comment by amulware** [comment]
@monday8am bump :P

**Comment by monday8am** [comment]
except for the use of ".this" :P , for me is ok. Good job!

**Pull request merged**